{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2021-42343",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "An issue was discovered in the Dask distributed package before 2021.10.0 for Python. Single machine Dask clusters started with dask.distributed.LocalCluster or dask.distributed.Client (which defaults to using LocalCluster) would mistakenly configure their respective Dask workers to listen on external interfaces (typically with a randomly selected high port) rather than only on localhost. A Dask cluster created using this method (when running on a machine that has an applicable port exposed) could be used by a sophisticated attacker to achieve remote code execution.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2021-42343",
				"url": "https://www.suse.com/security/cve/CVE-2021-42343"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1192072 for CVE-2021-42343",
				"url": "https://bugzilla.suse.com/1192072"
			}
		],
		"title": "SUSE CVE CVE-2021-42343",
		"tracking": {
			"current_release_date": "2023-02-15T03:37:16Z",
			"generator": {
				"date": "2023-02-15T03:37:16Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2021-42343",
			"initial_release_date": "2023-02-15T03:37:16Z",
			"revision_history": [
				{
					"date": "2023-02-15T03:37:16Z",
					"number": "2",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "2"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-distributed-2024.4.2-1.1",
								"product": {
									"name": "python310-distributed-2024.4.2-1.1",
									"product_id": "python310-distributed-2024.4.2-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python311-distributed-2024.4.2-1.1",
								"product": {
									"name": "python311-distributed-2024.4.2-1.1",
									"product_id": "python311-distributed-2024.4.2-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python312-distributed-2024.4.2-1.1",
								"product": {
									"name": "python312-distributed-2024.4.2-1.1",
									"product_id": "python312-distributed-2024.4.2-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python38-distributed-2022.1.0-1.1",
								"product": {
									"name": "python38-distributed-2022.1.0-1.1",
									"product_id": "python38-distributed-2022.1.0-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python39-distributed-2022.1.0-1.1",
								"product": {
									"name": "python39-distributed-2022.1.0-1.1",
									"product_id": "python39-distributed-2022.1.0-1.1"
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-distributed-2024.4.2-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-distributed-2024.4.2-1.1"
				},
				"product_reference": "python310-distributed-2024.4.2-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-distributed-2024.4.2-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-distributed-2024.4.2-1.1"
				},
				"product_reference": "python311-distributed-2024.4.2-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-distributed-2024.4.2-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-distributed-2024.4.2-1.1"
				},
				"product_reference": "python312-distributed-2024.4.2-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python38-distributed-2022.1.0-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python38-distributed-2022.1.0-1.1"
				},
				"product_reference": "python38-distributed-2022.1.0-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python39-distributed-2022.1.0-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python39-distributed-2022.1.0-1.1"
				},
				"product_reference": "python39-distributed-2022.1.0-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2021-42343",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-42343"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in the Dask distributed package before 2021.10.0 for Python. Single machine Dask clusters started with dask.distributed.LocalCluster or dask.distributed.Client (which defaults to using LocalCluster) would mistakenly configure their respective Dask workers to listen on external interfaces (typically with a randomly selected high port) rather than only on localhost. A Dask cluster created using this method (when running on a machine that has an applicable port exposed) could be used by a sophisticated attacker to achieve remote code execution.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"fixed": [
					"openSUSE Tumbleweed:python310-distributed-2024.4.2-1.1",
					"openSUSE Tumbleweed:python311-distributed-2024.4.2-1.1",
					"openSUSE Tumbleweed:python312-distributed-2024.4.2-1.1",
					"openSUSE Tumbleweed:python38-distributed-2022.1.0-1.1",
					"openSUSE Tumbleweed:python39-distributed-2022.1.0-1.1"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-42343",
					"url": "https://www.suse.com/security/cve/CVE-2021-42343"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1192072 for CVE-2021-42343",
					"url": "https://bugzilla.suse.com/1192072"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-distributed-2024.4.2-1.1",
						"openSUSE Tumbleweed:python311-distributed-2024.4.2-1.1",
						"openSUSE Tumbleweed:python312-distributed-2024.4.2-1.1",
						"openSUSE Tumbleweed:python38-distributed-2022.1.0-1.1",
						"openSUSE Tumbleweed:python39-distributed-2022.1.0-1.1"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 9.8,
						"baseSeverity": "CRITICAL",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-distributed-2024.4.2-1.1",
						"openSUSE Tumbleweed:python311-distributed-2024.4.2-1.1",
						"openSUSE Tumbleweed:python312-distributed-2024.4.2-1.1",
						"openSUSE Tumbleweed:python38-distributed-2022.1.0-1.1",
						"openSUSE Tumbleweed:python39-distributed-2022.1.0-1.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2021-10-27T12:45:22Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2021-42343"
		}
	]
}
